guides/v2.2/ext-best-practices/extension-coding/security-performance-data-bp.md

```diff
@@ -85,3 +85,19 @@ public function getCustomerCart()
     return $this->_cart;
 }
 ```
+
+## Use the appropriate area
+
+Make sure that your observer or plugin is declared only in the appropriate area:
+
+- [`adminhtml`]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Backend/etc/di.xml){:target="_blank"}
+- [`crontab`]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Cron/etc/di.xml){:target="_blank"}
+- [`frontend`]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Store/etc/di.xml){:target="_blank"}
+- [`graphql`]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/GraphQl/etc/di.xml){:target="_blank"}
```

Comment:
@eduard13 , do we have a GraphQl in 2.2?

Reply:
Good point 👍